packages/tpetra/core/test/PerformanceCGSolve/cg_solve_file.cpp

```diff
@@ -83,16 +82,6 @@ bool cg_solve (Teuchos::RCP<CrsMatrix> A, Teuchos::RCP<Vector> b, Teuchos::RCP<V
   p = Tpetra::createVector<ScalarType>(A->getRangeMap());
   Ap = Tpetra::createVector<ScalarType>(A->getRangeMap());
 
-  {
-    TimeMonitor t(*TimeMonitor::getNewTimer(replaceValsName));
-    int length = r->getLocalLength();
-    for(int i = 0;i<length;i++) {
-      x->replaceLocalValue(i,0);
```

Comment:
Just a note for further documentation- Even if this x vector hadn't already been initialized to zero, there should be no need to set x=0.  CG should run successfully with any initial guess x.